CHANGELOG.md

```diff
@@ -3,4 +3,10 @@
 ### Added
 
 - Self-hosted Gitlab Docker running on a EC2 instance.
-- Automated backups and restore
\ No newline at end of file
+- Automated backups and restore
+
+## [1.0.1] - 2022-12-30
```

Comment:
i would put last changes first to avoid misunderstandings

Reply:
Fixed, put the last changes in the first lines.